app/views/splashes/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class= 'splash-container'>
+<h1 class= 'home-heading'>Welcome to I-Spend</h1>
+<div class= 'home-signup'><%= link_to "Sign Up", new_user_registration_path%></div>
+<div class= 'home-signin'><%= link_to "Log In", new_user_session_path%></div>
+</div>
```

Comment:
- [ ] it would be better if you use the same Colors Typography: font face, size and weight. and Layout: composition and space between elements. in order to match design so please try to do that 
- You should make the background with the same gray color in the design 
- make the sign up and log in as blue buttons as in the design 

- The design 
<img width="281" alt="Screen Shot 2022-10-14 at 19 26 20" src="https://user-images.githubusercontent.com/61790587/195916465-1122923b-8a51-4a79-ad24-bb92092c25f7.png">
- Yours 
<img width="394" alt="Screen Shot 2022-10-14 at 19 26 43" src="https://user-images.githubusercontent.com/61790587/195916487-25cc88c9-e06e-41f3-810a-529097b0d8fe.png">